src/main/java/seedu/internbuddy/logic/commands/EditCommand.java

```diff
@@ -229,6 +234,14 @@ public Optional<Address> getAddress() {
             return Optional.ofNullable(address);
         }
 
+        public void setIsFavourite(Boolean isFavourite) {
+            this.isFavourite = isFavourite;
+        }
+
+        public Optional<Boolean> getIsFavourite() {
+            return Optional.ofNullable(isFavourite);
```

Comment:
Love the use of Java Coding Standards with isFavourite naming for the boolean, as well as a setter and getter to read and update the boolean values.